encoding/rs/encode.go

```diff
@@ -122,27 +111,31 @@ func (g *Encoder) MakeFrames(
 	return frames, indices, nil
 }
 
-// Encoding Reed Solomon using FFT
-func (g *Encoder) ExtendPolyEval(coeffs []fr.Element) ([]fr.Element, []fr.Element, error) {
-
+func (g *Encoder) PadPolyEval(coeffs []fr.Element) ([]fr.Element, error) {
 	if len(coeffs) > int(g.NumEvaluations()) {
-		return nil, nil, errors.New("the provided encoding parameters are not sufficient for the size of the data input")
+		return nil, fmt.Errorf("the provided encoding parameters are not sufficient for the size of the data input")
 	}
 
 	pdCoeffs := make([]fr.Element, g.NumEvaluations())
 	for i := 0; i < len(coeffs); i++ {
 		pdCoeffs[i].Set(&coeffs[i])
 	}
+	// Padding to GPU
 	for i := len(coeffs); i < len(pdCoeffs); i++ {
 		pdCoeffs[i].SetZero()
 	}
+	return pdCoeffs, nil
+}
 
-	evals, err := g.Fs.FFT(pdCoeffs, false)
+// Encoding Reed Solomon using FFT
+func (g *Encoder) ExtendPolyEval(coeffs []fr.Element) ([]fr.Element, error) {
+
+	evals, err := g.Fs.FFT(coeffs, false)
 	if err != nil {
-		return nil, nil, err
+		return nil, err
 	}
 
-	return evals, pdCoeffs, nil
+	return evals, nil
```

Comment:
Think you can remove this method here since it's been moved to `{cpu,gpu}/extend_poly.go`?